SWE574/core/templates/about2.html

```diff
@@ -0,0 +1,137 @@
+{% extends 'base.html' %}
```

Comment:
The only difference between this html and about.html must be the navbar. 
Navbar of this html should stay the same whereas the navbar of about.html must be inherited from navbar.html by using 
```
{% include 'navbar.html' %}
```